drivers/kscan/Kconfig.ft5336

```diff
@@ -14,14 +14,23 @@ if KSCAN_FT5336
 
 config KSCAN_FT5336_PERIOD
 	int "Sample period"
-	depends on !KSCAN_FT5336_INTERRUPT
 	default 10
 	help
-	  Sample period in milliseconds when in polling mode.
+	  Sample period in milliseconds when in polling or
+	  interrupt polling mode.
 
 config KSCAN_FT5336_INTERRUPT
 	bool "Enable interrupt"
 	help
 	  Enable interrupt support (requires GPIO).
 
+config KSCAN_FT5336_INTERRUPT_POLLING_MODE
+	bool "Enable interrupt polling mode"
+	depends on KSCAN_FT5336_INTERRUPT
+	default 10
```

Comment:
polling_mode is just a flag. this shouldn't be bool?
anyway period is defined above KSCAN_FT5336_PERIOD

Reply:
yes, you need to remove this.
usually, KConfig changes should have separate commit and c code files should have separate commit.